index.js

```diff
@@ -222,8 +233,27 @@ export default class QRCodeScanner extends Component {
       pendingAuthorizationView,
       cameraType,
     } = this.props;
+
+    if (!this.state.isCameraActivated) {
+      return (
+        <TouchableWithoutFeedback onPress={() => this._setCamera(true)}>
+          <View style={[styles.camera, styles.inactiveContainer]}>
+            <Text style={styles.inactiveLabel}>Tap to activate camera</Text>
```

Comment:
wouldn't it be better to have this component to the timeout message coming from the outside instead as a prop?

Reply:
Yes that would be good. I'll resubmit the PR